book/src/ui-configuration.md

```diff
@@ -1,37 +1,117 @@
-# Configuration
+# 📦 Installation
 
-Siren requires a connection to both a Lighthouse Validator Client and a Lighthouse Beacon Node.  
-To enable connection, you must generate .env file based on the provided .env.example
+Siren supports any operating system that supports container runtimes and/or NodeJS 18, this includes Linux, MacOS, and Windows. The recommended way of running Siren is by launching the [docker container](https://hub.docker.com/r/sigp/siren).
+
+## Version Requirement
+
+To ensure proper functionality, the Siren app requires Lighthouse v4.3.0 or higher. You can find these versions on the [releases](https://github.com/sigp/lighthouse/releases) page of the Lighthouse repository.
 
-## Connecting to the Clients
+## Configuration
+
+Siren requires a connection to both a Lighthouse Validator Client and a Lighthouse Beacon Node.  
 
 Both the Beacon node and the Validator client need to have their HTTP APIs enabled.  
-These ports should be accessible from Siren.  
+These ports should be accessible from Siren. This means adding the flag `--http` on both beacon node and validator client.
 
 To enable the HTTP API for the beacon node, utilize the `--gui` CLI flag. This action ensures that the HTTP API can be accessed by other software on the same machine.
 
 > The Beacon Node must be run with the `--gui` flag set.
 
-If you require accessibility from another machine within the network, configure the `--http-address` to match the local LAN IP of the system running the Beacon Node and Validator Client.
+## Running the Docker container (Recommended)
+
+The common usage is to run Siren at a client computer connecting to a server running the node. The following guide is to setup Siren at a client computer and connect it to the server via SSH, so that Siren can be accessed and viewed from the client computer's browser.
```

Comment:
this is 100% not the case anymore with the "new" dockerised setup. 
the preferred way to run it nowadays is to have the docker container as close to the BN and VC as possible, running all the time, with its https port exposed to local LAN. 
(this was the whole reason of the architectural overahaul)

Reply:
Revised in [2c6cd76](https://github.com/sigp/lighthouse/pull/6553/commits/2c6cd76a7e7580ebb69ca56510b448417ff91fc1)